### DISCUSSION TITLE
Using aeotec stick gen5

### DISCUSSION THREAD

**Discussion by mazmar** [zwave-js/zwave-js-ui#2233] [Q&A] [answered]
So I've just discovered I am using an obsolete version of aeotec z-stick ZW090 which does not support S2 nor SmartStart ...

What is totally weird is that some of my nodes show S2_Authenticated in ZwaveJs UI
![Screenshot 2022-02-03 at 12 38 21](https://user-images.githubusercontent.com/597771/152335857-6c5a1747-013f-4963-82b8-2b26a8d5b773.png)

but most of the nodes will never finish pairing with S2 and after inputing DSK it will resolve security as none ...
I currently have 22 nodes which 6 are shown as S2_Authenticated and others are none
How is that possible? Aeotec states that this adapter only supports S0

What are my options?

Should I go buy the z-stick gen5+ or z-stick gen7 (which is out of stock everywhere ...) and redo all of my nodes and HA setup?

Is there a way to replace a adapter without needing to include all devices again?

I guess i would need to remove the not secured devices anyway so any kind of backup/restore is pointless anyway as I only have 6 nodes showing S2_Authenticated. 

Is this all worth it? Should i be concerned about z-wave security? Its not like i expect anybody going around with sniffer and messing with my network.

I was thinking about running 2 zwavejs instances with second on raspberry-pi with shield 700series so any new device would be assigned to this network. But i wonder if that is viable for HA. I could run the second via MQTT discovery and the original via zwavejs ws port. 
That would separate them in HA. Right?

Currently i only have lights and sensors on my network but i plan to have a lock in my smart home. However i didn't find any good EU lock for Z-Wave so this might not even be an issue. Considering I have all exported to HomeKit i am waiting for locks with new home key implemented ...

**Comment by robertsLando** [comment]
@AlCalzone

**Comment by AlCalzone** [comment, marked as answer]
> I am using an obsolete version of aeotec z-stick ZW090 which does not support S2

S2 is handled by Z-Wave JS, not the stick.

> nor SmartStart

Do you know there is a firmware update which adds support for this? https://help.aeotec.com/support/solutions/articles/6000252294-z-stick-gen5-v1-02-firmware-update

> and redo all of my nodes and HA setup?

This shouldn't be necessary, but if you decide to go that route, you can backup and restore the controller memory with zwavejs2mqtt.

> but most of the nodes will never finish pairing with S2 and after inputing DSK it will resolve security as none

There's likely something else going on. If you want to investigate that, [open an issue](https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=&template=bug_report.yml) and provide the requested information.

> I was thinking about running 2 zwavejs instances with second on raspberry-pi with shield 700series so any new device would be assigned to this network

I wouldn't split the mesh into two, especially when it is small. Otherwise you're gonna have trouble with communication.

**Reply by mazmar** [reply]
unfortunately i have the non + version of the Gen5 so FW ugprade will not help with smartstart.

good point about having separate networks being trouble. I guess groups would not work ... 

I will try getting debug logs from the S2 inclusions but it happened long time ago and i didn't download the logs then.

I guess restoring memory can only be done to the same z-stick? Or could i backup gen5 and restore it to gen7? Or maybe even different vendor? Aeotec is out of stock for Gen7 but i was able to get some ZShield for raspberry pi. I will test that

Thanks

**Reply by mazmar** [reply]
This page shows that S2 should not be supported on my z-stick https://aeotec.com/z-wave-home-automation/development-kit-pcb.html


<img width="1397" alt="Screenshot 2022-02-05 at 10 23 16" src="https://user-images.githubusercontent.com/597771/152636064-6d8d40b3-c1f6-437e-acf7-1dc19dd6e607.png">

**Reply by AlCalzone** [reply]
> unfortunately i have the non + version of the Gen5 so FW ugprade will not help with smartstart.

That firmware update is specifically for the non+ version.

> I guess restoring memory can only be done to the same z-stick? Or could i backup gen5 and restore it to gen7?

Zwave-js automatically converts the memory format. But that requires the 500 series stick firmware to be recent enough.
For the gen5 stick, that is the aforementioned version 1.2

**Reply by mazmar** [reply]
Omg that is so stupidly placed in the support page. I totally saw that long ago and skipped it as irrelevant as it is in gen5+ section. I updated today. Will rework my network later this week. Thanks
